src/test/integration/web_interface/test_filter.py

```diff
@@ -60,9 +60,3 @@ def test_render_analysis_tags_fix(frontend):
         output = render_analysis_tags(tags).replace('\n', '').replace('    ', ' ')
     assert 'badge-primary' in output
     assert '> wow<' in output
-
-
-def test_render_analysis_tags_bad_type():
```

Comment:
What made this test useless?

Reply:
It aimed at provoking a ValueError by submitting an Int as key but it didn't cause an error (maybe because of the updated version). Testing this doesn't even make sense, since the tested function doesn't even handle such errors.